src/main/java/fashionable/simba/yanawaserver/domain/Court.java

```diff
@@ -0,0 +1,33 @@
+package fashionable.simba.yanawaserver.domain;
+
+import fashionable.simba.yanawaserver.error.InvaildCourtNameException;
+
+import java.util.UUID;
+
+public class Court {
+    private final UUID id;
+    private final String name;
+    private final String location;
+
+    public Court(UUID id, String name, String location) {
+        if (name.isEmpty() || name.length() > 15) {
+            throw new InvaildCourtNameException();
+        }
```

Comment:
String 값이 null 인지 확인할 필요도 있습니다. 그리고 name이 공백일 경우에도 테스트가 통과할 것 같아요. 
이런 경우 공백을 제거해서 빈 값인지를 확인할 필요가 있는데 `string.trim().isEmpty()` 메서드를 사용해도 되고 같은 기능인 `isBlank()`를 사용해도 좋습니다.

```suggestion
        if (name == null || name.trim().isEmpty() || name.length() > 15) {
            throw new InvaildCourtNameException();
        }
```